src/OpenLoco/GameCommands/UpdateOwnerStatus.cpp

```diff
@@ -0,0 +1,28 @@
+#include "../CompanyManager.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    static loco_global<CompanyId, 0x009C68EB> _updatingCompanyId;
```

Comment:
I think we want to minimise declarations of loco_global, so you can make these getters/setters in `gameState`, and then you call the getter/setter from here

Reply:
Done!